### PR TITLE
Added inclusion filter for bytes type

### PR DESCRIPTION
**Pull request by serejkaaa512**
Fixes [this](https://github.com/prisma/prisma/issues/8854).

### CODE REVIEW

**Comment by 0xdeafbeef** [comment]
Finally!

**Comment by serejkaaa512** [comment]
> [CLA assistant check](https://cla-assistant.io/prisma/prisma-engines?pullRequest=2418) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/prisma/prisma-engines?pullRequest=2418) before we can accept your contribution.
> 
> **serejkaaa512** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/prisma/prisma-engines?pullRequest=2418) it.

I have signed it. What to do to remove this?

**Comment by dpetrick** [comment]
> I have signed it. What to do to remove this?

Read the entire message.

File: query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/bytes_filter.rs [hunk 1]
```diff
@@ -63,6 +63,11 @@ mod bytes_filter_spec {
           @r###"{"data":{"findManyTestModel":[{"id":1}]}}"###
         );
 
+        insta::assert_snapshot!(
```
Comment (dpetrick) [inline query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/bytes_filter.rs:66]: Please add a negation test for completeness as well.

**Review by dpetrick — CHANGES_REQUESTED** [review]

File: query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/bytes_filter.rs (hunk 1 above)
Comment (serejkaaa512) [reply query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/bytes_filter.rs:66]: done

**Review by dpetrick — APPROVED** [review]
We can't merge _this_ PR (edit: to master) because internal CI is not running on forks. I will replay your changes on a branch and credit you.

However, for legal reasons the CLA must be signed and green before I can do anything.

**Comment by serejkaaa512** [comment]
> We can't merge _this_ PR (edit: to master) because internal CI is not running on forks. I will replay your changes on a branch and credit you.
> 
> However, for legal reasons the CLA must be signed and green before I can do anything.

Signed finally.

**Pull request merged**